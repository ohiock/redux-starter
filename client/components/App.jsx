import React, { PropTypes } from 'react';
import { Link, IndexLink } from 'react-router';

const App = props => (
  <div>
    <ul>
      <li><IndexLink to="/">Home</IndexLink></li>
      <li><Link to="/counter">Counter</Link></li>
    </ul>
    <br />
    { props.children }
  </div>
);

App.propTypes = {
  children: PropTypes.element.isRequired,
};

export default App;
