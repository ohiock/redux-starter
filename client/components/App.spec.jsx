import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line

import App from './App';

describe('App', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <App />,
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
