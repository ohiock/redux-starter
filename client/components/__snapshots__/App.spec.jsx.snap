// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <ul>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Counter
      </a>
    </li>
  </ul>
  <br />
</div>
`;

exports[`renders correctly 1`] = `
<div>
  <ul>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Counter
      </a>
    </li>
  </ul>
  <br />
</div>
`;
